src/tests/unit/tests/common/components/cards/__snapshots__/result-section.test.tsx.snap

```diff
@@ -1,101 +1,161 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ResultSection renders with shouldAlertFailureCount = <false> 1`] = `
-<div
-  className="result-section-class-name resultSection"
-  data-automation-id="result-section"
->
-  <HeadingElementForLevel
-    headingLevel={2}
+<DocumentFragment>
+  <div
+    class="result-section-class-name resultSection"
+    data-automation-id="result-section"
   >
-    <ResultSectionTitle
-      containerClassName="result-section-class-name"
-      deps={
-        {
-          "getNextHeadingLevel": [Function],
-        }
-      }
-      sectionHeadingLevel={2}
-      shouldAlertFailuresCount={false}
-      titleSize="title"
+    <h2
```

Comment:
We could mock the `HeadingElementForLevel` component to cut down on superfluous HTML here, but the amount added is small enough that I could take it or leave it. Up to you!

Reply:
Yes, we have mocked **HeadingElementForLevel** component as per suggestion.